doc/user/USAGE.md

```diff
@@ -59,7 +59,7 @@ Here is how the plugins are searched (by precedence order):
 1. Search the static plugins
 2. Consider the option given is a full path
 3. Search in the paths specified in `F3D_PLUGINS_PATH` environment variable
-4. Search in the same directory than `f3d.exe` (Windows only)
+4. Search in a directory relative to the F3D application ( `./` on Windows, `../lib` otherwise).
```

Comment:
Is there any reason we are putting plugin DLLs in "bin" instead of "lib" on Windows?